lib/home/library/play_video.dart

```diff
@@ -30,15 +30,13 @@ class _PlayVideoState extends State<PlayVideo> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        appBar: AppBar(title: Text(widget.videoName)),
-        body: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: [
-            CustomVideoPlayer(
-              customVideoPlayerController: _customVideoPlayerController,
-            ),
-          ],
-        ));
+      appBar: AppBar(title: Text(widget.videoName)),
+      body: Center(
+        child: CustomVideoPlayer(
+          customVideoPlayerController: _customVideoPlayerController,
+        ),
+      ),
+    );
```

Comment:
If a `Row` or `Column` has only 1 child, you can boost performance by swapping it out for a different widget.

(I meant to add this back when I reviewed #49 but then forgot to do so 🙃)